src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceTestTransformer.java

```diff
@@ -426,13 +429,25 @@ private void addUnitTestImports(InterfaceContext context) {
         typeTable.saveNicknameFor("java.util.concurrent.ExecutionException");
         break;
       case HTTP:
-        typeTable.saveNicknameFor("com.google.api.gax.rpc.StatusCode.Code");
-        typeTable.saveNicknameFor("com.google.api.gax.rpc.testing.FakeStatusCode");
-        typeTable.saveNicknameFor("com.google.api.gax.httpjson.MockHttpService");
+        typeTable.saveNicknameFor("com.google.api.gax.httpjson.ApiMessage");
+        typeTable.saveNicknameFor("com.google.api.gax.httpjson.ApiMethodDescriptor");
+        typeTable.saveNicknameFor("com.google.api.gax.httpjson.testing.MockHttpService");
```

Comment:
This depends on https://github.com/googleapis/gax-java/pull/493 which is still WIP, right?

Reply:
This depends on https://github.com/googleapis/gax-java/pull/492, which is a PR that can/should be reviewed in parallel with this toolkit PR.

Reply:
Although yes good catch I'm reverting the MockHttpService package to `com.google.api.gax.httpjson`.